packages/wrangler/src/pages/utils.ts

```diff
@@ -18,3 +20,14 @@ export function isUrl(maybeUrl?: string): maybeUrl is string {
 		return false;
 	}
 }
+
+let realTmpdirCache: string | undefined;
+/**
```

Comment:
biggest nit ever for formatting
```suggestion

/**
```